build/Targets/Versions.props

```diff
@@ -10,14 +10,14 @@
 
   <PropertyGroup>
     <!-- This is the assembly version of Roslyn from the .NET assembly perspective. It should only be revved during significant point releases. -->
-    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.9.0</RoslynAssemblyVersionBase>
+    <RoslynAssemblyVersionBase Condition="'$(RoslynAssemblyVersion)' == ''">2.11.0</RoslynAssemblyVersionBase>
```

Comment:
Awaiting a decision between `2.11.0` and `3.0.0` so I went with the probably-safer answer for now until things calm down.

Reply:
This is probably for the best anyways prior to us scouting out the potential Willow breaks.